library/packages/src/lib/y2packager/product.rb

```diff
@@ -256,5 +268,18 @@ def status?(*statuses)
         statuses.include?(res["status"])
       end
     end
+
+    # Return product's resolvable properties
+    #
+    # Only the 'name' and 'version' will be used to find out whether the
+    # product properties, ignoring the architecture, vendor or any other
```

Comment:
I do not get this sentence. What means that "whether the product properties" I miss something there